python/jupyter-notebooks/ohsome_green_region.ipynb

```diff
@@ -0,0 +1,833 @@
+      "text/plain": [
+       "<Figure size 864x576 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "df_density = pd.read_pickle(\"./df_density.pkl\")\n",
+    "\n",
+    "val = {'region':['Vancouver', 'Halifax', 'Edmonton', 'Ottawa'], \n",
+    "       'mean': [204006.8824050633, 223159.97221518986, 28518.290506329115, 234835.11246835446], \n",
+    "       'median': [260064.895, 296758.975, 32828.065, 316338.055], 'min':[614.07, 0.0, 0.0, 1942.07], \n",
+    "       'max':[319494.75, 324080.16, 49962.98, 330267.47]}\n",
+    "val_density = pd.DataFrame(val)\n",
+    "\n",
+    "#df_density.plot(kind='box', figsize=(12,10), fontsize=9)\n",
+    "display (val_density)\n",
+    "val_density.plot.barh(0, figsize=(12,8))\n",
+    "\n",
+    "#The table was created by searching for the mean, median, minimum and maximum values of each region.\n",
+    "#You can do it with numpy, like so:\n",
+    "#Ottawa, Ontario:\n",
+    "#Oarray = df_density['Ottawa']\n",
+    "#OttMean = np.mean(Oarray)\n",
+    "#OttMedian = np.median(Oarray)\n",
+    "#OttMin = np.min(Oarray)\n",
+    "#OttMax = np.max(Oarray)\n",
+    "#display(\"Ottawa\")\n",
+    "#display(OttMean, OttMedian, OttMin, OttMax) #-> to look at the values"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "formed-morning",
+   "metadata": {},
+   "source": [
+    "#### _Above you can see the output DataFrame 'val_density' with some statistics for the density dataset, below that it was visualized in a barchart_"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "golden-soviet",
+   "metadata": {},
+   "source": [
+    "Apart from Ottawa there is no display of a minimum value in the barchart above. That is due to the fact that Edmonton and Halifax both started at zero in the beginning of the examined timespan but also because the initial value of Metro Vancouver was much too small to be visibly display in the graphic.\n",
```

Comment:
```suggestion
    "Apart from Ottawa there is no display of a minimum value in the barchart above. That is due to the fact that Edmonton and Halifax both started at zero in the beginning of the examined timespan but also because the initial value of Metro Vancouver was much too small to be visible in the graphic.\n",
```